3rdparty/openvx/include/openvx_hal.hpp

```diff
@@ -0,0 +1,227 @@
+#ifndef OPENVX_HAL_HPP_INCLUDED
+#define OPENVX_HAL_HPP_INCLUDED
+
+#include <cstdio>
+#include "opencv2/core/hal/interface.h"
+#include "VX/vx.h"
+#include "VX/vxu.h"
+#include <string>
+#include <iostream>
+
+//===================================
+// utility
+// ...
+
+#include <chrono>
```

Comment:
Currently we do not have access to the TickMeter class from the custom HAL library. This (_Tick_) class was used during performance issues investigation, it is not used in actual code.